include/boot_target/android-boot.jinja2

```diff
@@ -0,0 +1,9 @@
+- boot:
+    docker:
+      image: {{url_docker}}
+    method: fastboot
+    prompts:
+    - 'root@(.*):/#'
+    - 'console:/'
+    timeout:
+      minutes: {{timeout_boot}}
```

Comment:
we have a variable called TARGET_BOOT_TIMEOUT for this

Reply:
will update to use TARGET_BOOT_TIMEOUT